tools/read_schema.py

```diff
@@ -0,0 +1,69 @@
+import sys
+import json
+import torch
+import numpy as np
+import torch.distributed as dist
+
+import dgl
+
+def read_json(json_file):
+    with open(json_file) as schema:
+        val = json.load(schema)
+
+    return val
+
+def get_node_types(schema): 
+    nid_ranges = schema['nid']
+    nid_ranges = {key: np.array(nid_ranges[key]).reshape(1,2)
+                    for key in nid_ranges}
+    ntypes = [(key, nid_ranges[key][0,0]) for key in nid_ranges]
+    ntypes.sort(key=lambda e: e[1])
```

Comment:
the returned ntype needs to be sorted based on some order. can you write comments to explain what order you use?